### DISCUSSION TITLE
ready for release 0.13?

### DISCUSSION THREAD

**Discussion by cheese1** [wfx/teg#49] [General]
hi

are we ready for a release of version 0.13?
long time since last updates, maybe those are settled now.

**Comment by wfx** [comment]
No because we have some tiny bugs (like the translation).

**Reply by cheese1** [reply]
sorry for being ignorant, could you point me to the translation-errors, maybe i could help with that

**Reply by wfx** [reply]
You are welcome, take a look on issue 47

**Comment by wfx** [comment]
Done :)
